test/unit/model/define.test.js

```diff
@@ -195,7 +195,7 @@ describe(Support.getTestDialectTeaser('Model'), () => {
       }).to.throw('Invalid definition for "part.name", "notNull" validator is only allowed with "allowNull:false"');
     });
 
-    // TODO: once we have centralized logging, enable this again
+    // TODO!: once we have centralized logging, enable this again
```

Comment:
Here

Reply:
Fixed in https://github.com/sequelize/sequelize/pull/14505/commits/0f62d00fec8df539c1ca4086fc2abf917019b28c